frontend/src/v4/modules/sagas.ts

```diff
@@ -94,6 +95,7 @@ export default function* rootSaga() {
 		fork(containersSaga),
 		fork(federationsSaga),
 		fork(revisionsSaga),
+		fork(TicketsSagas),
```

Comment:
Fixed

Reply:
Sorry to be a pain, but I think it should be Saga, not Sagas.
Well, really I think it would make sense if they were all Sagas, but lets just go with what's there currently